### ISSUE TITLE
[Improvement]: Terminal improvements

### ISSUE THREAD

**Issue by ayntk-ai** [coollabsio/coolify#2651]
### Description

Note: The terminal is not added at the moment but will be soon: https://github.com/coollabsio/coolify/pull/2586

### Minimal Reproduction (if possible, example repository)

Some improvement ideas for the terminal system could be:
- [ ]  A full log of all the commands we have run in the terminal (with automatic log rotation and deletion) -> So we can keep track of what we have run on our system.
- [ ]  A snippet store -> so we can store most used commands as snippets and easily execute them with one click (not have to retype them every time) -> For example `docker system prune` on the server
- [ ]  Run commands on multiple containers at the same time -> could be achieved with the tags system --> But maybe not that smart for security?

### Version

-

**Comment by LEstradioto** [comment]
Oh I can see more clearly now about these ideas.

About number 3, If I understand correctly, this could be some kind of Cron system right? Like, setup some bash scripting or whatever, and run it periodically or by startup button

**Comment by ayntk-ai** [comment]
> Oh I can see more clearly now about these ideas.
> 
> About number 3, If I understand correctly, this could be some kind of Cron system right? Like, setup some bash scripting or whatever, and run it periodically or by startup button

Yes. Also for 3, when you add tags, there is a way to re-deploy all containers with those tags, maybe the tags functionality can be leveraged for terminal as well.
